pkg/build/build.go

```diff
@@ -146,7 +146,10 @@ func (b *core) Steps() []Step {
 
 func (b *core) createDirectories() error {
 	c := b.config
-	return fs.Mkdirs(c.Paths.TargetDir(), c.Paths.ZipDir(), c.Paths.MetaDir)
+	if err := fs.MkdirEmpty(c.Paths.TargetDir()); err != nil {
+		return err
+	}
+	return fs.Mkdirs(c.Paths.ZipDir(), c.Paths.MetaDir)
```

Comment:
This change ensures that the target directory is empty at the start of the build process. This is pretty important for reproducibility. It wasn't noticed before, since we were using uniquely-named temporary directories for most builds. However, now we are building directly into `dist/.../...`, the issue came to light.